backends/advanced/src/advanced_omi_backend/routers/modules/system_routes.py

```
@@ -128,6 +129,53 @@ async def delete_all_user_memories(current_user: User = Depends(current_active_u
     return await system_controller.delete_all_user_memories(current_user)
 
 
+# Chat Configuration Management Endpoints
+
+@router.get("/admin/chat/config", response_class=Response)
+async def get_chat_config(current_user: User = Depends(current_superuser)):
+    """Get chat configuration as YAML. Admin only."""
+    try:
+        yaml_content = await system_controller.get_chat_config_yaml()
+        return Response(content=yaml_content, media_type="text/plain")
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)